lib/suggestions.js

```diff
@@ -32,6 +34,31 @@ function resolveModule(textEditor, module) {
 
         return path.join(basedir, module)
     }
+
+    // Allow to resolve module directories to the current directory and searched for modules
+    if (resolveModulesDirectories.length > 0) {
+      if (path.extname(module) == '') {
+          module += '.js'
+      }
+
+      var filename;
+
+      atom.project.getPaths().forEach(projectPath => {
+          if (textEditor.getPath().indexOf(projectPath) !== -1) {
+              resolveModulesDirectories.forEach(moduleDirectory => {
+                  let resolvePath = path.join(path.join(projectPath, moduleDirectory), module);
```

Comment:
fyi, `path.join` accepts as many parameters as you need, so this could be simplified to

    let resolvePath = path.join(projectPath, moduleDirectory, module);